### PR TITLE
0.9.0 Release

### PR DESCRIPTION
**Pull request by lbrendanl**
This contains the necessary changes for the 0.9.0 release.  This includes:

- Updating docs based on new contract (feel free to ignore changes under docs/, this is all autogenerated).
- Updated old samples to use new 0.9.0 library.
- Added a new sample that show cases the UI namespace.

### CODE REVIEW

File: Samples/UINamespace/uiNamespace.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    <!-- jQuery -->
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" >
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" ></script>
+
+    <!-- Extensions Library (this will be hosted on a CDN eventually) -->
+    <script src="../../lib/tableau-extensions-0.9.0.js"></script>
```
Comment (sdesmond46) [inline Samples/UINamespace/uiNamespace.html]: Should we create some sort of tableau-extensions-0.latest.js version of the bootstrap so we don't always need to modify all of these?

File: Samples/UINamespace/uiNamespace.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      // settings in sync.  This event will be triggered any time a setting is
+      // changed for this extension, in the parent or popup (i.e. when settings.saveAsync is called).
+      tableau.extensions.settings.addEventListener(tableau.TableauEventType.SettingsChanged, (settingsEvent) => {
+        updateExtensionBasedOnSettings(settingsEvent.newSettings)
+      });
+    });
+  });
+
+  function configure() {
+    const popupUrl = 'http://localhost:8765/Samples/UINamespace/uiNamespaceDialog.html';
```
Comment (sdesmond46) [inline Samples/UINamespace/uiNamespace.js]: Should this use window.location to dynamically get this url?

File: Samples/UINamespace/uiNamespace.html (hunk 1 above)
Comment (lbrendanl) [reply Samples/UINamespace/uiNamespace.html]: Yeah that's a good suggestion, I'll do that.

File: Samples/UINamespace/uiNamespace.js (hunk 2 above)
Comment (lbrendanl) [reply Samples/UINamespace/uiNamespace.js]: Good idea, will add.

**Pull request merged**